src/frontend/apps/impress/src/utils/string.ts

```diff
@@ -1,5 +1,5 @@
 export const isValidEmail = (email: string) => {
   return !!email.match(
```

Comment:
Maybe we can test instead of cast into boolean for better lisibility ?
wdyt ? 

```suggestion
const EMAIL_REGEX =
  /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z\-0-9]{2,}))$/;
  return EMAIL_REGEX.test(email);

```